tests/unit-tests/jobs/models/docker/jobConnector-test.js

```diff
@@ -212,4 +212,34 @@ describe('Docker job connector tests', function () {
             }
         });
     });
-});
\ No newline at end of file
+
+    describe('Delete all containers', () => {
+        it('Delete 3 containers', async () => {
+            listContainersStub.resolves([
+                { Names: ['predator-runner.0'] },
+                { Names: ['predator-runner.1'] },
+                { Names: ['predator-runner.2'] }]);
```

Comment:
add another entry to array resolved with `Names: ['someJob.0']` and expect same results